admin-dev/themes/default/template/header.tpl

```diff
@@ -288,19 +288,19 @@
                   {$active = "active"}
                   {if $show_new_orders}
                     <li class="nav-item {$active}">
-                      <a class="nav-link" data-toggle="tab" data-type="order" href="#orders-notifications" role="tab" id="orders-tab">{l s='Latest orders' d='Admin.Navigation.Header'}<span id="orders_notif_value"></span></a>
+                      <a class="nav-link" data-toggle="tab" data-type="order" href="#orders-notifications" role="tab" id="orders-tab">{l s='Latest orders' d='Admin.Navigation.Header'}<span class="notif-counter"></span></a>
```

Comment:
Hi @davidglezz I agree with you that classes are better than IDs however I'm afraid we cannot remove these identifiers.
If a module uses selectors to target these identifiers, this will break this module behavior, so this would be a BC break.

We are not against BC breaks but they must serve a specific purpose (like fixing a bug). Introducing a BC break that might be destructive for some modules relying on it does not look like a purpose high enough.

However if you can provide valid reasons for removing these identifiers 🤔 maybe they can be removed. But without valid reasons, we have to keep them at least until next major version of PrestaShop 😞

Reply:
Agree with matks, if these changes are trivial, we could accept it and tag it as a BC break, as well as document it inside the next release, but if not, then it's impossible on a minor release :/

Reply:
By the way, you can still add classes and use them inside you'r JS, while you can keep Ids only for retrocompatibility

Reply:
Agree with both.
I have kept the ids so as not a BC break.
The ids were only used by this js, but could be used by a third-party module.

If BC will be break, I think that is preferable a own PR only for it.